testdata/project-v3-multigroup/main.go

```diff
@@ -69,7 +70,9 @@ func init() {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
+	var probeAddr string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&probeAddr, "probe-address", ":8081", "The address the probe endpoint binds to.")
```

Comment:
```suggestion
	flag.StringVar(&probeAddr, "probe-bind-address", ":8081", "The address the probe endpoint binds to.")
```
To follow up the same metrics standard.

Reply:
Done